opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/TestSpringWebTracing.java

```diff
@@ -63,15 +77,23 @@ public void testControllerTracing() {
 
   @Test
   public void testRestTemplateTracing() {
-    restTemplate.getForEntity("http://www.example.com", String.class);
+    restTemplate.getForEntity(getUrl("/notTraced"), String.class);
     await().until(() -> mockTracer.finishedSpans().size() == 1);
-    assertEquals(1, mockTracer.finishedSpans().size());
+    List<MockSpan> mockSpans = mockTracer.finishedSpans();
+    assertEquals(1, mockSpans.size());
+    assertEquals(Tags.SPAN_KIND_CLIENT, mockSpans.get(0).tags().get(Tags.SPAN_KIND.getKey()));
```

Comment:
Sounds good then. Perhaps it would be worth adding a comment explaining this ? Something like 

```java
// when using the REST template, the OT Spring Integration creates a client span, and this span is what we expect here
```